tests/unknown-op/unknown-00.txt

```diff
@@ -1,2 +1,2 @@
-brun --strict -c '(a (q 0x0000) (q ()))'
+brun --strict -c '(a (q . 0x000) (q . ()))'
```

Comment:
I'm not sure why these worked without having the quote updated. But I re-generated these to cover more interesting cases, with the maximum cost multiplier as well as invalid cost multipliers (5 bytes wide)

Reply:
If you're changing these, can you change them to not use `a` (since its semantics are changing)? You can just go `(0x0000 (q . ()))` for example.